app/lib/github-auth.ts

```diff
@@ -18,9 +18,13 @@ export class GitHubAppAuth {
   private config: GitHubAppConfig;
 
   constructor() {
+    const rawPrivateKey = process.env.GITHUB_PRIVATE_KEY || "";
+
+    const privateKey = rawPrivateKey.replace(/\\n/g, "\n");
+
     this.config = {
       appId: GITHUB_APP_ID,
-      privateKey: process.env.GITHUB_PRIVATE_KEY || "",
+      privateKey,
```

Comment:
I'm still getting this error:

Error merging PR: Error: secretOrPrivateKey must be an asymmetric key when using RS256

Can you print the first 4 and last 4 chars of the private key, as well as the length of the key